bottles/backend/managers/sandbox.py

```diff
@@ -50,50 +50,6 @@ def __init__(
         self.share_gpu = share_gpu
         self.__uid = os.environ.get("UID", "1000")
 
-    def __get_bwrap(self, cmd: str):
```

Comment:
I am not sure if bwrap will be used in the Snap package I am working on with some friends at Canonical. Just a note, anyway we can re-introduce the changes in case.

Reply:
Won't they be able to run `flatpak-spawn` in the sandbox anyway?